### PR TITLE
Fix Orca error message: Orca supports Rollup

### PR DESCRIPTION
**Pull request by vraghavan78**
Fixing error message. Orca supports rollup when the error states otherwise. [#117665973].

@xinzweb @d @oarap @hsyuan please take a look.

### CODE REVIEW

**Comment by danielgustafsson** [comment]
LGTM

File: src/backend/gpopt/translate/CTranslatorUtils.cpp [hunk 1]
```diff
@@ -1469,9 +1469,9 @@ CTranslatorUtils::PdrgpbsGroupBy
 		GPOS_RAISE(gpdxl::ExmaDXL, gpdxl::ExmiQuery2DXLUnsupportedFeature, GPOS_WSZ_LIT("Group by clause"));
 	}
 
-	// grouping sets
 	if (1 != gpdb::UlListLength(plGroupClause))
```
Comment (xinzweb) [inline src/backend/gpopt/translate/CTranslatorUtils.cpp]: Can that be 0? Shall that be `1 <`?

**Comment by xinzweb** [comment]
Any test updates to verify the new error message? If not, would you please add one? Thanks.

**Comment by hsyuan** [comment]
+1

File: src/backend/gpopt/translate/CTranslatorUtils.cpp (hunk 1 above)
Comment (vraghavan78) [inline src/backend/gpopt/translate/CTranslatorUtils.cpp]: no

**Pull request closed without merging**